pipeline/pipeline-notes.rst

```diff
@@ -62,20 +121,17 @@ Running the pipeline
 First, check out the source repository and grab the (...large) initial data
 sets::
 
- cd /mnt
- git clone https://github.com/ged-lab/2012-paper-diginorm.git
- cd 2012-paper-diginorm
+ cd ${HOME}
 
- curl -O https://s3.amazonaws.com/public.ged.msu.edu/2012-paper-diginorm/pipeline-data-new.tar.gz
- tar xzf pipeline-data-new.tar.gz
+ git clone https://github.com/ged-lab/2012-paper-diginorm.git --branch non-root-install
 
 Now go into the pipeline directory and install Prokka & run the pipeline.  This
 will take 24-36 hours, so you might want to do it in 'screen' (see
 http://ged.msu.edu/angus/tutorials-2011/unix_long_jobs.html). ::
 
- cd pipeline
+ cd 2012-paper-diginorm/pipeline
  bash install-prokka.sh
- make /usr/local/share/khmer
+ make 
```

Comment:
@mr-c I don't understand why this line is not working. Before I added lines 90 and 91, I had the problem `no module screed`. Are lines 90 and 91 incorrect ways to install screed? If not, I now have this error after make:
```
miguel@cloud-server-02:~/2012-paper-diginorm/pipeline$ make
load-into-counting.py -k 20 -N 4 -x 1e8 genome-reads.fa.ct genome-reads.fa
make: load-into-counting.py: Command not found
make: *** [genome-reads.fa.ct] Error 127
```

This makes me think this is a screed problem still.